pkg/cmd/controller/controller_build.go

```diff
@@ -1342,7 +1343,7 @@ func CreateReportTargetURL(templateText string, params ReportParams) string {
 	return buf.String()
 }
 
-func toScmStatusAndDescription(activity *v1.PipelineActivity) (string, string) {
+func toScmStatusDescriptionRunningStages(activity *v1.PipelineActivity) (string, string, string) {
```

Comment:
Getting close where I would create a struct and return it, three strings :-(

Reply:
That is fair. Lemme do that.